src/compositemark/boxplot.ts

```diff
@@ -169,70 +161,68 @@ export function normalizeBoxPlot(spec: GenericUnitSpec<Encoding<Field>, BOXPLOT
             type: continuousAxisChannelDef.type
           },
           ...nonPositionEncoding,
-          ...midTickAndBarSizeChannelDef,
-          color: {value : 'white'}
+          ...sizeMixins,
+          color: {value : 'white'} // FIXME: allow customizing this color
```

Comment:
Actually we can easily fix this now by removing this line and make 
`config.boxMid.color` = `"white"` by default.  

(And test a bit if it really works.)